### ISSUE TITLE
Reverse dependency graph returns internal server error

### ISSUE THREAD

**Issue by srijs** [rust-lang/crates.io#1839]
Hi!

Trying to view the reverse dependency graph of the [`crc32fast`](https://crates.io/crates/crc32fast) crate exhibits a weird behaviour (empty page, title says "Crate Not Found").

The network inspector shows that the API route is returning a 500 status code:

```
https://crates.io/api/v1/crates/crc32fast/reverse_dependencies?page=1&per_page=10
```

Happy to help fixing it, but due to the opaque status code it's not clear to me what needs fixing in the first place :)

**Comment by carols10cents** [comment]
Wow, this was a fun one to track down. Explanation and a fix over in #1846 !

**Comment by srijs** [comment]
Awesome, thanks for investigating and fixing this so quickly!
